README.md

```diff
@@ -1,34 +1,32 @@
-# notion-blog
+# notion-md-gen
 
-`notion-blog` allows you to use Notion as a CMS for pages built with hugo. You can use it as a cli or even automate your blog repo to update itself with the Github Action.
+`notion-md-gen` allows you to use Notion as a CMS for pages built with any static site generators. You can use it as a cli or even automate your blog repo to update itself with the Github Action.
 
 
 ## Requisites
 
 - Notion database for your articles.
 - Notion API secret token.
-- Hugo powered blog.
+- A blog by any static site generators.
+
+## Setup
+```bash
+brew install notion-md-gen
+```
 
 ## Usage
 
 ### CLI
 
-The cli shows the executable flags when using flag `—help`.
-
 ```bash
-$> notion-blog —help
+cd your-blog-dir
+notion-md-gen
 ```
 
-### Binary
-
-The binary looks for a config file called `notionblog.config.json` in the directory where it is executed. You can see the example config in [notionblog.config.json](notionblog.config.json).
```

Comment:
Maybe keeping some information about the config file, as when executing you don't get any information about it and the program simply fails.

Reply:
Here I plan to implement a logic that automatically generates a default configuration file

Reply:
recovered, give up the original plan for the time being